x/qgb/README.md

```diff
@@ -110,56 +360,30 @@ if err != nil {
 }
 ```
 
-- The universal nonce was not incremented correctly by 1:
+- The universal nonce wasn't incremented correctly by 1:
 
 ```golang
 if k.CheckLatestAttestationNonce(ctx) && k.GetLatestAttestationNonce(ctx)+1 != nonce {  
    panic("not incrementing latest attestation nonce correctly")  
 }
 ```
 
-## Verification command
-
-The QGB verification command is part of the `celestia-appd` binary. It allows the user to verify that a set of shares has been posted to a specific QGB contract.
-
-```shell
-$ celestia-appd verify --help          
-                              
-Verifies that a transaction hash, a range of shares, or a blob referenced by its transaction hash were committed to by the QGB contract
-
-Usage:
-  celestia-appd verify [command]
-
-Available Commands:
-  blob        Verifies that a blob, referenced by its transaction hash, in hex format, has been committed to by the QGB contract.
-  shares      Verifies that a range of shares has been committed to by the QGB contract
-  tx          Verifies that a transaction hash, in hex format, has been committed to by the QGB contract
-
-Flags:
-  -h, --help   help for verify
-
-Global Flags:
-      --home string         directory for config and data (default "/home/midnight/.celestia-app")
-      --log_format string   The logging format (json|plain) (default "plain")
-      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
-      --trace               print out full stack trace on errors
+### Hooks initialization panic
 
-Use "celestia-appd verify [command] --help" for more information about a command.
+When initializing the QGB hooks, if the QGB store key is not setup correctly, the state machine will [panic](https://github.com/celestiaorg/celestia-app/blob/0629c757ef35a24187a8d7a4c706c7cdc894c8b6/x/qgb/keeper/hooks.go#L14-L19):
 
+```golang
+// if startup is mis-ordered in app.go this hook will halt the chain when
+// called. Keep this check to make such a mistake obvious
+if k.storeKey == nil {
+panic("hooks initialized before GravityKeeper")
```

Comment:
adding this, but tbh probably not worth the extra round of reviews lol

used 3 " " instead of 4 since the rest are using 3
```suggestion
   panic("hooks initialized before GravityKeeper")
```

Reply:
I will handle it in a subsequent PR, thanks :+1:

Reply:
https://github.com/celestiaorg/celestia-app/pull/1976